uv/lib/dependabot/uv/file_updater/compile_file_updater.rb

```diff
@@ -37,6 +37,7 @@ class CompileFileUpdater
           "pip._internal.exceptions.InstallationSubprocessError: Getting requirements to build wheel exited with 1",
           String
         )
+        PYTHON_VERSION_REGEX = T.let(/--python-version[=\s]+(?<version>\d+\.\d+)/, Regexp)
```

Comment:
Should this also capture the `patch` version? (Is it even allowed?)

I ignore how common that will be, but if an user pins to something like `3.11.2`, this code here will try to use `3.11` which may yield incorrect results